docs/changelogs/v0.16.md

```diff
@@ -85,6 +86,33 @@ $ ipfs config Routing.Methods --json '{
 
 ```
 
+### 🌍 WebTransport new experimental Transport
+
+A new feature of [`go-libp2p`](https://github.com/libp2p/go-libp2p/releases/tag/v0.23.0) is [WebTransport](https://github.com/libp2p/go-libp2p/issues/1717).
+
+For now it is **disabled by default** and considered **experimental**.
+If you find issues running it please [report them to us](https://github.com/ipfs/kubo/issues/new).
+
+In the future we will change Kubo to listen on WebTransport by default (and listening) for anyone already listening `QUIC`, when we think it is stabilized.
+
+This transport will allow in-browser nodes to contact Kubo nodes using a new
+fancy API which gets around mixed content issues, the end goal is for js-ipfs
+running inside a browser to access the IPFS network as a client.
+
+#### How to enable WebTransport
```

Comment:
Can you add these examples into the documentation too?

Reply:
Done